### PR TITLE
Form Integrations

### PR DESCRIPTION
**Pull request by Soare-Robert-Daniel**
Providers
- [x] Mailchimp
- [x] Sendinblue

### CODE REVIEW

**Comment by Soare-Robert-Daniel** [comment]
@HardeepAsrani, can you also check the integration with Mailchimp? When testing the form, you need to provide a "good" email. Emails like `a@a.com` are considered fake or invalid.

**Comment by Soare-Robert-Daniel** [comment]
@HardeepAsrani, this is the current look of the integration tabs ( _the API key field has been set to password type for this screenshot only_ )

image

File: inc/server/class-form-server.php [hunk 1]
```diff
@@ -58,8 +58,39 @@ public function register_routes() {
 				),
 			)
 		);
+
+
+		register_rest_route(
+			$namespace,
+			'/mailchimp',
```
Comment (HardeepAsrani) [inline inc/server/class-form-server.php:65]: You can use one 'newsletter' route and pass the provider as an argument so we don't need to make a new rest route each time we want to integrate with a new service. @Soare-Robert-Daniel

**Comment by Soare-Robert-Daniel** [comment]
@HardeepAsrani, I have grouped the function in separated classes.  It will prevent the Form class from growing too chaotic.

**Comment by Soare-Robert-Daniel** [comment]
@irinelenache, can you test this?

**Comment by Soare-Robert-Daniel** [comment]
All changes are now in #793.

**Pull request closed without merging**